src/background.js

```diff
@@ -557,6 +559,7 @@ const setup = {
       runAddonBypassPref ||
       runAddonDoorhangerDecision === "UIOk" ||
       runAddonDoorhangerDecision === "enabled" ||
+      runAddonPreviousTRRMode === 2 ||
```

Comment:
This doesn't work for users who stopped at mode 0 (after being in mode 2) in the previous version.

Reply:
Resolved and verified. Great catch.